### DISCUSSION TITLE
[Question] Any way to mark occurrences of multiple strings in a file?

### DISCUSSION THREAD

**Discussion by pgp** [geany/geany#3446] [General]
With Notepad++, one can mark all occurrences multiple times (each time for different strings), and all the sets of occurrences remain highlighted till one clicks the related Clear button from the search menu.
Is there any way to do the same in geany? By default, when marking for a string, the previous marked occurrences (if any), are not highlighted anymore.

**Comment by elextr** [comment]
No, only a single search can be in progress at any one time.
